package.json

```diff
@@ -1,157 +1,21 @@
-    "@types/d3": "^6.7.1",
-    "@types/jest": "^25.2.1",
-    "@types/lingui__core": "^2.7.1",
-    "@types/lingui__macro": "^2.7.4",
-    "@types/lingui__react": "^2.8.3",
-    "@types/ms.macro": "^2.0.0",
-    "@types/multicodec": "^1.0.0",
-    "@types/node": "^13.13.5",
-    "@types/qs": "^6.9.2",
-    "@types/react": "^17.0.2",
-    "@types/react-dom": "^17.0.1",
-    "@types/react-helmet": "^6.1.5",
-    "@types/react-redux": "^7.1.16",
-    "@types/react-router-dom": "^5.0.0",
-    "@types/react-router-hash-link": "^2.4.5",
-    "@types/react-virtualized-auto-sizer": "^1.0.0",
-    "@types/react-window": "^1.8.2",
-    "@types/rebass": "^4.0.7",
-    "@types/styled-components": "^5.1.25",
-    "@types/testing-library__cypress": "^5.0.5",
-    "@types/ua-parser-js": "^0.7.35",
-    "@types/wcag-contrast": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^4",
-    "@typescript-eslint/parser": "^4",
-    "craco-workbox": "^0.2.0",
-    "cypress": "^7.7.0",
-    "env-cmd": "^10.1.0",
-    "eslint": "^7.11.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-better-styled-components": "^1.1.2",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^4.0.0",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "firebase": "^9.1.3",
-    "graphql": "^15.5.0",
-    "graphql-request": "^3.4.0",
-    "inter-ui": "^3.13.1",
-    "jest-fetch-mock": "^3.0.3",
-    "jest-styled-components": "^7.0.5",
-    "npm-run-all": "^4.1.5",
-    "patch-package": "^6.4.7",
-    "polyfill-object.fromentries": "^1.0.1",
-    "prettier": "^2.2.1",
-    "qs": "^6.9.4",
-    "react": "^17.0.1",
-    "react-appzi": "^1.0.4",
-    "react-confetti": "^6.0.0",
-    "react-cosmos": "^5.6.6",
-    "react-dom": "^17.0.1",
-    "react-ga4": "^1.4.1",
-    "react-inlinesvg": "^2.3.0",
-    "react-is": "^17.0.2",
-    "react-router-hash-link": "2.4.0",
-    "react-scripts": "^4.0.3",
-    "serve": "^11.3.2",
-    "start-server-and-test": "^1.11.0",
-    "text-encoding-polyfill": "^0.6.7",
-    "typechain": "^8.1.0",
-    "typescript": "^4.4.3",
-    "ua-parser-js": "^0.7.28",
-    "use-count-up": "^2.2.5",
-    "use-resize-observer": "^8.0.0",
-    "web-vitals": "^2.1.0",
-    "webpack-bundle-analyzer": "^4.5.0",
-    "workbox-core": "^6.1.0",
-    "workbox-precaching": "^6.1.0",
-    "workbox-routing": "^6.1.0",
-    "yarn-audit-fix": "^9.2.1"
-  },
-  "resolutions": {
-    "@walletconnect/ethereum-provider": "1.7.1",
-    "react-error-overlay": "6.0.9",
-    "@types/react": "^17.0.2"
-  },
+  "description": "CoW Swap",
+  "main": "index.js",
+  "author": "",
+  "license": "ISC",
   "scripts": {
+    "start": "craco start",
+    "build": "yarn i18n:compile && craco build && yarn writeVersion",
+    "test": "NODE_PATH=src/custom craco test --env=jsdom",
     "start:default": "craco start",
```

Comment:
do we need it if we have the `start` already?

Reply:
Yep, we don't need it, thanks!